lib/common/http1client.c

```diff
@@ -465,7 +478,9 @@ static void on_head(h2o_socket_t *sock, const char *err)
         }
         h2o_buffer_consume(&client->sock->input, rlen);
         if (client->sock->input->size == 0) {
-            h2o_timer_link(client->super.ctx->loop, client->super.ctx->io_timeout, &client->super._timeout);
+            if (! h2o_timer_is_linked(&client->super._timeout)) {
+                h2o_timer_link(client->super.ctx->loop, client->super.ctx->io_timeout, &client->super._timeout);
+            }
```

Comment:
Thank you, but I just applied `clang-format` at https://github.com/h2o/h2o/pull/3316/commits/03f8d32e29a69f3ed3813031ecded75b4e1e9a91 since your suggestion seems to be removing `!` :)

Or did you actually want to remove braces?

Reply:
Of course you are correct!